.buildbot.sh

```diff
@@ -33,6 +33,15 @@ for dir in hybrid hybrid/ddc_compartment_switching syscall-restrict; do
     popd
 done
 
+echo "Checking that all the Morello purecap examples build..."
+platform='morello-purecap'
+for dir in morello-c64-secure; do
```

Comment:
Perhaps I'm missing something here; why not add `morello-c64-secure` to the list of directories on line 18 (at time of writing), which also runs purecap tests?

I wonder whether it's because this example is Morello-specific and has no `riscv64-purecap` variant to run, in which case maybe [the earlier comment from @0152la](https://github.com/capablevms/cheri-examples/pull/96/files#r1574520120) could be resolved by stating that only riscv64 examples are being built (rather than removing `purecap` altogether in the echo'd text), and building morello targets here by duplicating the list of examples in the earlier `for` loop and removing `morello-purecap` from the list of platforms it builds.

Reply:
I went with your suggestion to run a separate set of tests for each platform: [73485f1](https://github.com/capablevms/cheri-examples/pull/96/commits/73485f1e63609c6be691b323fbb89dfb5b01b056)

Another approach is to iterate through every directory, and build everything that has a makefile. That'd be much simpler, but I worried that an incorrectly-named (or missing) makefile might never be spotted.

Reply:
Looks great to me! Cheers! I think some extra indentation might have snuck into the `morello-purecap` bit on two lines, but not exactly sure and it doesn't matter anyway, so marking resolved 👍️